### PR TITLE
GitHub Actions migration added examples and images to tabs

### PR DESCRIPTION
**Pull request by csidyel**
## What
GitHub Actions migration page: fixed some text typos and similar. Added images to the tabs. Redid the complete examples.

## Checklist: 
 

Use a checklist to confirm:
- [x] The branch is up-to-date with the main branch.
- [x] Update follows the docs [style guide](../../docs/docs-contributing/STYLE_GUIDE.md).
- [x] Changes have been tested locally.

### CODE REVIEW

**Comment by TomFern** [comment]
@csidyel does the build work on your machine? When I try to start the main branch in this repo I get this error:

```
Error: MDX compilation failed for file "/Users/tom/r/semaphore-christian/docs/docs/getting-started/migration/github-actions.md"
Cause: Unexpected closing tag `</Tabs>`, expected corresponding closing tag for `<TabItem>` (70:1-70:46)
Details:
{
  "column": 1,
  "message": "Unexpected closing tag `</Tabs>`, expected corresponding closing tag for `<TabItem>` (70:1-70:46)",
  "line": 74,
  "name": "74:1-74:8",
  "place": {
    "start": {
      "line": 74,
      "column": 1,
      "offset": 2241,
      "_index": 125,
      "_bufferIndex": 0
    },
    "end": {
      "line": 74,
      "column": 8,
      "offset": 2248,
      "_index": 126,
      "_bufferIndex": -1
    }
  },
  "reason": "Unexpected closing tag `</Tabs>`, expected corresponding closing tag for `<TabItem>` (70:1-70:46)",
  "ruleId": "end-tag-mismatch",
  "source": "mdast-util-mdx-jsx"
}
```

**Comment by TomFern** [comment]
I found the fix, actually there are two things you need to fix the build 

On `docs/getting-started/migration/github-actions.md`:
- Line 260: replace `value="semaphore"` with `value="ui"`
- Line 74: add `</TabItem>` before `</Tabs>`

That should build the page. Overall I think **it's a great tutorial** and the tabs really helped. My only observation is that I think the UI screenshot should show the same commands as in the YAML. Otherwise the image and the Semaphore YAML dont' really coincide.

You can highlight the relevant YAML like this:

- Highlight range:

    ```yaml
    checkout
    # highlight-start
    sem-version node 20
    # highlight-end
    ```

- Or one line:

      ```yaml
      checkout
      # highlight-next-line
      sem-version node 20
      ```

File: docs/docs/getting-started/migration/github-actions.md [hunk 1]
```diff
@@ -40,87 +40,93 @@ jobs:
...
+    commands:
+      - checkout
 ```
 
 </TabItem>
+<TabItem value="ui" label="Semaphore Editor">
+
+![Checking out the repository](./img/checkout.jpg)
+
 </Tabs>
```
Comment (TomFern) [inline docs/docs/getting-started/migration/github-actions.md:75]:
```suggestion
</TabItem>
</Tabs>
```

File: docs/docs/getting-started/migration/github-actions.md [hunk 2]
```diff
@@ -129,91 +135,131 @@ The following example caches Gems in a Ruby project:
...
+jobs:
+  - name: my_job
+    commands:
+      - artifact pull workflow test.log
+```
+
+See [artifacts](../../using-semaphore/artifacts) for more details.
+
+</TabItem>
+<TabItem value="semaphore" label="Semaphore YAML">
```
Comment (TomFern) [inline docs/docs/getting-started/migration/github-actions.md]:
```suggestion
<TabItem value="ui" label="Semaphore Editor">
```

File: docs/docs/getting-started/migration/github-actions.md [hunk 3]
```diff
@@ -40,87 +40,93 @@ jobs:
...
 
-In Semaphore, we use the [artifact](../../reference/toolbox#artifact) command to download and upload files to the artifact store.
+Semaphore uses [sem-version](../../reference/toolbox#sem-version) to activate or switch language versions in the CI environment. 
 
-The following command stores `test.log` from any job:
+The following example activates Ruby v3.3.4, any commands after the example run on this Ruby version.
 
-```shell
-artifact push workflow test.log
+```yaml
```
Comment (TomFern) [inline docs/docs/getting-started/migration/github-actions.md:104]: The YAML and the UI screenshot do not have the same commands. I think the tabs Semaphore YAML and Semaphore Editor should always coincide.

There are other examples of this below.

**Review by TomFern — CHANGES_REQUESTED** [review]
@csidyel thank you so much for the PR. Amost there! There are a couple of notes to revise.

**Comment by csidyel** [comment]
@TomFern I addressed all the requested changes and uploaded new images. Please, check it out now.

**Review by csidyel — COMMENTED** [review]
Issues addressed.

**Review by TomFern — APPROVED** [review]
@csidyel perfect thank you! Great work.

Tip for future screenshots: disable Grammarly and spell checking in the textboxes. This is just a detail, nothing major, but it's nice to remove all distractions.

**Pull request merged**